bindings/java/src/main/java/org/ucx/jucx/Bridge.java

```diff
@@ -4,6 +4,8 @@
  */
 package org.ucx.jucx;
 
+import org.ucx.jucx.Worker.CompletionQueue;
```

Comment:
package name should be all in all lower case to avoid conflict with the names of classes or interfaces.

Reply:
The package is:
```java
org.ucx.jucx
```
This is only an import to Worker's inner class - CompletionQueue.

Reply:
i see, thx